pkg/detectors/auth0oauth/auth0oauth.go

```diff
@@ -37,20 +37,24 @@ func (s Scanner) Keywords() []string {
 // FromData will find and optionally verify Auth0oauth secrets in a given set of bytes.
 func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (results []detectors.Result, err error) {
 	dataStr := string(data)
-
 	clientIdMatches := clientIdPat.FindAllStringSubmatch(dataStr, -1)
 	clientSecretMatches := clientSecretPat.FindAllStringSubmatch(dataStr, -1)
 	domainMatches := domainPat.FindAllStringSubmatch(dataStr, -1)
+	uniqueDomainMatches := make(map[string]struct{})
+	for _, m := range domainMatches {
+		if len(m) > 1 {
+			uniqueDomainMatches[strings.TrimSpace(m[1])] = struct{}{}
+		}
```

Comment:
This is superfluous:
```suggestion
	uniqueDomainMatches[m[1]] = struct{}{}
```

Reply:
is it always?

Reply:
I'd say so.

Reply:
https://pkg.go.dev/regexp#Regexp.FindAllStringSubmatch guess you're right